catalog/plugin/catalog-plugin-federation-replication/src/main/java/ddf/catalog/federation/layered/replication/RestReplicatorPlugin.java

```diff
@@ -192,7 +193,7 @@ private String transform(Metacard m, WebClient client) throws PluginExecutionExc
 
         BinaryContent binaryContent;
         try {
-            binaryContent = transformer.transform(m, null);
```

Comment:
I didn't do that because that will create an empty, immutable map. I'm not certain if a plugin could be expecting to modify that map in some way.